node/api/consensus.go

```diff
@@ -25,6 +25,13 @@ type ConsensusHeadersGET struct {
 	BlockID types.BlockID `json:"blockid"`
 }
 
+// ConsensusBlocksGet wraps a types.Block and adds an id field.
+type ConsensusBlocksGet struct {
+	BlockID     types.BlockID     `json:"id"`
+	BlockHeight types.BlockHeight `json:"height"`
```

Comment:
is the difference between the field name in the struct and the field name in the json intentional?

Reply:
Yes, because the embedded `Block` already has a `ID` method. That's why I changed it to `BlockID`. `BlockID` and `Height` looks weird so I changed the `Height` to `BlockHeight`. 
For the JSON response `id` and `height` make sense though.

Reply:
Have you found a way to get `BlockHeight` from `ID`?   Besides, how long would it take for you to finish this? I need finish my job before next Monday. So I'm working in a new branch to get those data support.  I couldn't figure out except using the `current_height - 1`, but I  still need  find the `current_height` in my code.  If there is a better solution, plz let me know.

Reply:
@hupengbuaa this PR should be pretty much done. I'm just waiting for @DavidVorick or @lukechampine to approve and merge it.

Reply:
Oh, thx. I see.